public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -16,16 +19,47 @@ import { isReceiverUsed } from '../../utils/alertmanager';
 import { isVanillaPrometheusAlertManagerDataSource } from '../../utils/datasource';
 import { makeAMLink } from '../../utils/misc';
 import { extractNotifierTypeCounts } from '../../utils/receivers';
+import { initialAsyncRequestState } from '../../utils/redux';
 import { ProvisioningBadge } from '../Provisioning';
 import { ActionIcon } from '../rules/ActionIcon';
 
 import { ReceiversSection } from './ReceiversSection';
 
+interface ReceiverErrorProps {
+  errorCount: number;
+}
+function ReceiverError({ errorCount }: ReceiverErrorProps) {
+  const styles = useStyles2(getStyles);
+  return (
+    <div className={styles.warning}>
+      <Stack alignItems="center">
+        <Icon name="exclamation-triangle" />
+        <div className={styles.countMessage}>{`${errorCount} ${pluralize('error', errorCount)}`}</div>
+      </Stack>
+    </div>
+  );
+}
+interface ReceiverHealthProps {
+  errorsByReceiver: number;
+}
+
+function ReceiverHealth({ errorsByReceiver }: ReceiverHealthProps) {
+  return errorsByReceiver > 0 ? <ReceiverError errorCount={errorsByReceiver} /> : <div>OK</div>;
```

Comment:
I think it would be nice if we used a `<Badge>` component for the health status, you can add a tooltip and an icon to that component too!

https://developers.grafana.com/ui/latest/index.html?path=/story/data-display-badge--basic&args=text:Healthy;color:green;icon:check;tooltip:No+issues+detected

<img width="155" alt="image" src="https://user-images.githubusercontent.com/868844/181217605-05665f09-e3f8-411e-9627-22f4354512cf.png">

Reply:
done! 
<img width="1653" alt="Screenshot 2022-07-27 at 12 18 40" src="https://user-images.githubusercontent.com/33540275/181231678-0afe0e88-b90e-4169-8c14-92385bf59ab5.png">